components/micro-gateway-core/src/main/ballerina/src/gateway/filters/throttle_filter.bal

```diff
@@ -279,7 +279,10 @@ function isAPILevelThrottled(string apiContext, string? apiVersion, ConditionalT
     boolean stopOnQuota;
     string apiThrottleKey = apiContext;
     if (apiVersion is string) {
-        apiThrottleKey += ":" + apiVersion;
+        if (!apiThrottleKey.endsWith('/' + apiVersion)) {
```

Comment:
Do we test with and without conditional throttling polices

Reply:
tested and ok